frb_example/with_flutter/ios/Runner/bridge_generated.h

```diff
@@ -2,89 +2,139 @@
 #include <stdint.h>
 #include <stdlib.h>
 
+typedef int64_t DartPort;
+
+typedef bool (*DartPostCObjectFnType)(DartPort port_id, void *message);
+
+#if !defined(DEFINE_WASM)
```

Comment:
Curious, is it possible we really define `DEFINE_WASM` here? b/c if so, this header will be an empty file.

If `#if !defined(DEFINE_WASM)` is always true, maybe we can remove it (probably by modifying cbindgen config?)

Reply:
Might be better to silence warnings instead of putting this #define in.